src/components/Calendar/Toolbar/CustomEventDialog/DaySelector.tsx

```diff
@@ -11,6 +11,12 @@ interface DaySelectorProps {
 interface DaySelectorState {
     days: boolean[];
 }
+
+function detectMobile() {
```

Comment:
nit: rename this function to `isMobile` so that it reads more naturally in the conditional later.